### ISSUE TITLE
Proofreaders

### ISSUE THREAD

**Issue by iandoug** [source-foundry/legible#13]
Stumbled across this:

https://www.pgdp.net/c/faq/font_sample.php

Free to download.

Looks like the tilde escaped their attention, or they were happy with a swung-dash look.

**Comment by chrissimpkins** [comment]
Interesting, I've never heard of such a thing.  Feel free to post any character combinations that you come up with as a new issue report!

**Comment by iandoug** [comment]
Okay  I see these in their sample text: (apart from others we know like 1 l, O 0 etc.
R B
spaces missing after . or , or wrong side of other punctuation
F P
a e ?
ri == n
tie == the
h b

Following suggestions probably more of a problem in variable rather than fixed-pitch fonts

from IMDB: rn == m. That one frequently catches me. Need to zoom in to check.

others:
cl == d
LI == U  (seen today on hand-painted sign)

**Comment by chrissimpkins** [comment]
:+1:  please add individual concerns as new issue reports using title format that currently is in place with other character issues.  This will provide the opportunity for us to (1) maintain and easily search for a record of these issues (2) discuss them through the use of the visual diff tool.

**Comment by iandoug** [comment]
The reason I didn't add them separately as issues is because I'm not convinced that they are all a problem. Have attached screenshots of list above in my curated list of "least likely to confuse" monospace fonts. The issues of letters next to each other are not such a problem, as for example, in Arial or OpenSans. Also the design of mono lower-l and upper-I 
![spectacle tj5385](https://user-images.githubusercontent.com/12673886/28442086-a92c87c0-6dae-11e7-8b96-be5b5ecaef9b.png)
![spectacle tj6988](https://user-images.githubusercontent.com/12673886/28442108-e11e1482-6dae-11e7-9644-ba921005c9c8.png)

generally prevents such merge confusions.

So from the list above, I would suggest only these:
FP
hb
maybe rn/m
and maybe ae depending on how font displays them.

Fonts below are 11.5pt, normal and bold italic.

**Comment by chrissimpkins** [comment]
👍   Suggest examining them across a range of sizes to see if that assessment holds true as the box decreases in size.  This is where lack of space amplifies problems (and good instruction sets for hinting can significantly help!).  The problems tend to lie in the sub-10 range, particularly as you get down to 8 and below.  I generally inspect across a range from 6 - 15 when proofing glyphs and working on manual hinting in Hack. 

Agree with each of those combinations that you pointed out at this size.  Post them as new issue reports and we can begin to explore each as a separate analysis.

**Comment by iandoug** [comment]
Is it viable to code at point sizes below 10? I'm not a spring chicken any more but text that size is really just adding another unnecessary complication to the programming process, if you need to strain to see the words.
I understand from a font designer point of view you would want a comprehensive solution, but just seems a waste to me trying to get perfect shapes at unusable sizes... :-)

**Comment by chrissimpkins** [comment]
Yes many of our users use those ranges.  I have used down to 8 in the  past.  Below that and I get too much eye strain.  There are those out there who like to have as much visible code on the screen as possible...
